tools/ci_build/github/azure-pipelines/stages/set_packaging_variables_stage.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  IsReleaseBuild: false
+  PreReleaseVersionSuffixString: 'none'
+  PreReleaseVersionSuffixNumber: 0
+stages:
+- stage: Setup
+  jobs:
+  - job: Set_Variables
+    pool:
```

Comment:
Avoid using the shared machine pools, which are accessible by general public. 
You may resolve it in another PR when you have time.